letsencrypt/plugins/manual.py

```diff
@@ -26,13 +26,17 @@
 class Authenticator(common.Plugin):
     """Manual Authenticator.
 
-    .. todo:: Support for `~.challenges.DVSNI`.
-
+    This plugin requires user's manual intervention in setting up a HTTP
+    server for solving SimpleHTTP challenges and thus does not need to be
+    run as a privilidged process. Alternatively shows instructions on how
+    to use Python's built-in HTTP server and, in case of HTTPS, openssl
+    binary for temporary key/certificate generation.
     """
     zope.interface.implements(interfaces.IAuthenticator)
     zope.interface.classProvides(interfaces.IPluginFactory)
+#    hidden = True
 
-    description = "Manual Authenticator"
+    description = "Manually Edit Your Configuration"
```

Comment:
This is likely to be the most controversial part of this PR, but personally, I don't like this description. I would prefer something like "Manually configure and run a simple Python webserver".